report/main.Rmd

```diff
@@ -0,0 +1,313 @@
+
+In this dataset, the five intervention methods across sites could be transformed into four indicator variables, representing the presence and absence of Discount, Additional Messaging, Calorie Display, Exercise Display. The pre-intervension period and follow-up period are treated as baseline reference observations. Variables that would be included in the LME model would be day of week, four intervention indicators and duration into the intervention, clustered by site. The model development process involves selection of fixed effect and random effect parameters, typically guided by statistical tests like the log-likelihood test. 
+
+```{r lmem,echo=FALSE}
+
+```
+   
+
+## 5. Conclusions
+
+The recommended statistical process for assessing the impact of strategies to promote zero-calorie beverages over their sugary alternatives involves both exploratory and formal analyses. An exploratory data analysis will help identify underlying patterns in the data, such as correlation and missingness. The formal analysis is recommended to include three models: interrupted time series analysis, generalized estimating equations, and the linear mixed-effects model. Each of these models is capable of handling time-series and longitudinal data. Results from these models can be tested and compared for security. These analyses will answer if the data may indicate an impact on beverage sales by various labeling and discount strategies. 
+   
+   
+   
+
+## 6. References
+   
+
+- [\underline{\textcolor{blue}{Interrupted Time Series Analysis}}](https://rpubs.com/mbounthavong/MEPS_tutorial_6_itsa)
+- [\underline{\textcolor{blue}{Getting Started with Generalized Estimating Equations}}](https://library.virginia.edu/data/articles/getting-started-with-generalized-estimating-equations#:~:text=Generalized%20estimating%20equations%2C%20or%20GEE,(i.e.%2C%20model%20coefficients).)
+- Linear Mixed Effect Model Tutorials:
+  + [\underline{\textcolor{blue}{LMEM tutorial (illinois.edu)}}](https://jontalle.web.engr.illinois.edu/MISC/lme4/bw_LME_tutorial.pdf)
+  + [\underline{\textcolor{blue}{Chapter 17: Mixed Effects Modeling (uic.edu)}}](https://ademos.people.uic.edu/Chapter17.html)
+  + [\underline{\textcolor{blue}{Introduction to Linear Mixed Models (ucla.edu)}}](https://stats.idre.ucla.edu/other/mult-pkg/introduction-to-linear-mixed-models/)
+  + [\underline{\textcolor{blue}{Mixed Effects Models (bodywinter.com)}}](https://bodo-winter.net/tutorial/bw_LME_tutorial1.pdf)
+  + [\underline{\textcolor{blue}{Mixed Effects, lme4 Tutorial}}](https://ourcodingclub.github.io/tutorials/mixed-models/)
+  + [\underline{\textcolor{blue}{Fitting Linear Mixed-Effects Models Using lme4}}](https://cran.r-project.org/web/packages/lme4/vignettes/lmer.pdf)
+
+- ASDA Teaching Recordings:
+  + [\underline{\textcolor{blue}{Exploratory Data Analysis}}](https://community.grad.ubc.ca/gps/event/22318)
```

Comment:
These links cannot be accessed by people without a CWL ID. Therefore, I'm unsure if using them as references would be truly helpful.